doc/source/examples/03_modeling/export_design.mystnb

```diff
@@ -0,0 +1,164 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Exporting designs
+
+After a design has been created, users are typically interested in using bringing
+the design into a CAD tool for further development. This notebook demonstrates
+how to export a design to the various supported CAD formats.
+
+## Creating a design
+
+A simple design is created for demonstration purposes. This design consists of
+a set of rectangular pads with a circular hole in the center.
+
+```{code-cell} ipython3
+from ansys.geometry.core import Modeler
+from ansys.geometry.core.math import UNITVECTOR3D_X, UNITVECTOR3D_Y, Point2D
+from ansys.geometry.core.sketch import Sketch
+
+# Instantiate the modeler - this will connect to the service
+modeler = Modeler()
+
+# Create a new design
```

Comment:
```suggestion
# Create a design
```
You can't create an "old" design, so the "new" is unnecessary. Ansys doc (and software) is filled with "Create a new...."